### PR TITLE
add 'from' keyword

### PR DESCRIPTION
**Pull request by Gheoan**

### CODE REVIEW

File: TypeScript.tmLanguage [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (zhengbli) [inline TypeScript.tmLanguage]: Why is this change?

Comment (Gheoan) [inline TypeScript.tmLanguage]: No idea. PackageDev generates it like that. Maybe because I'm on Windows. I will manually remove it.

**Comment by Gheoan** [comment]
@zhengbli rebased and rady to merge

**Comment by zhengbli** [comment]
@Gheoan Thanks!

**Pull request merged**